h2o-core/src/main/java/hex/Model.java

```diff
@@ -213,6 +213,10 @@ public void setTrain(Key<Frame> train) {
       this._train = train;
     }
 
+    public void setValid(Key<Frame> valid) {
```

Comment:
Unused

Reply:
@michalkurka yeah, I wanted to keep it as it should have been there before.... like `setTrain()`.  Don't want to do a refactoring to make it used on the other hand.